libnavigation-core/src/main/java/com/mapbox/navigation/core/history/model/HistoryEventSetRoute.kt

```diff
@@ -14,7 +14,7 @@ import com.mapbox.api.directions.v5.models.DirectionsRoute
  * @param waypoints the coordinates for this route
  */
 class HistoryEventSetRoute internal constructor(
-    val directionsRoute: DirectionsRoute,
+    val directionsRoute: DirectionsRoute?,
```

Comment:
What's driving this change?

Reply:
don't know, it's from yours commit

Reply:
Ah, that's because Nav Native can return a null route.

Reply:
👍 , yeah i was about to [change this too](https://github.com/mapbox/mapbox-navigation-android/pull/4601/files#diff-76ce9cbef0d51b7f1bac812adf4f28cba6e43003aadcfd4c4d78f35ed5ef5a62L163)

the route is empty when the event represents "clear route"